src/mmds/src/token.rs

```diff
@@ -99,9 +102,16 @@ impl TokenAuthority {
             cipher: TokenAuthority::create_cipher(&mut file)?,
             num_encrypted_tokens: 0,
             entropy_pool: file,
+            aad: "".to_string(),
         })
     }
 
+    /// Set Additional Authenticated Data to be used for
+    /// encryption and decryption of the session token.
+    pub fn set_aad(&mut self, instance_id: &str) {
+        self.aad = format!("microvmid={}", instance_id);
```

Comment:
Is the formatting necessary ? To me it looks like it's being just being passed to the encryption fn.

Reply:
Formatting allows us to pass a string like: `microvmid=<instance_id>` instead of just `<instance_id`> to the encryption fn.